integ-test/src/test/java/org/opensearch/sql/legacy/TestsConstants.java

```diff
@@ -53,6 +53,7 @@ public class TestsConstants {
   public final static String TEST_INDEX_BEER = TEST_INDEX + "_beer";
   public final static String TEST_INDEX_NULL_MISSING = TEST_INDEX + "_null_missing";
   public final static String TEST_INDEX_CALCS = TEST_INDEX + "_calcs";
+  public final static String TEST_INDEX_MULTI_NESTED= TEST_INDEX + "_multi_nested";
```

Comment:
```suggestion
  public final static String TEST_INDEX_MULTI_NESTED = TEST_INDEX + "_multi_nested";
```

Reply:
This will be fixed in a non draft PR as there are many checkstyle failures with this PoC.